package.json

```
@@ -30,7 +30,7 @@
   },
   "peerDependencies": {
     "eslint": "^6.0.0 || ^7.0.0",
-    "typescript": "^3.7.5"
+    "typescript": "^3.7.5 || ^4"
```

Please use full semver to be consistent:
```suggestion
    "typescript": "^3.7.5 || ^4.0.0"
```